.devcontainer/develop_on_codespaces.rst

```diff
@@ -0,0 +1,94 @@
+.. _develop_on_codespaces:
+
+
+Develop on Codespaces
+=====================
+
+`Codespaces <codespaces_>`_ is a virtual development environment provided by GitHub.
+You can launch a container which all the required tools and start to work
+in couple of minutes.
+This is an easy way to get started with PyMAPDL development without going
+through the process of setting your environment.
+
+.. warning:: `Codespaces <codespaces_>`_  are not free, but they have a generous
+   free allowance per month.
+   After that, you must pay according to `GitHub pricing <github_pricing_>`_.
+   You can check your billing details in your `GitHub account settings` under
+   `Billing and plans` and `Plans and usage`.
+
+
+How to start
+------------
+
+To start a Codespace environment you must go to the repository page
+`PyMAPDL <pymapdl_repo_>`_, and click on the ``<> Code`` green button,
+and select the tab ``Codespaces``.
+There you can click on `+` to open a default Codespace environment.
```

Comment:
```suggestion
To start a Codespace environment, go to the
`PyMAPDL repository <pymapdl_repo_>`_, click the green **Code** button,
and select the **Codespaces** tab.
Then, click the **+** button to open a default Codespace environment.
```
Here, I knew for sure that you were referencing UI elements. Always use bold for them.

Reply:
I will try to remember!